Alternative docs (#52)
fox-it/dissect-docs: docs/source/install.rst

Install
=======

**Installing Dissect is easy**. The preferred way of installing Dissect is with ``pip``.
The install procedure is described below.

System requirements
~~~~~~~~~~~~~~~~~~~

These are recommended system requirements.
Dissect may still work on older systems but we do not actively
test on those.

* Operating system versions: Linux 4+, macOS 11+, Windows 8+
* Python 3.9 - 3.12
* Processor: 64-bit 1 gigahertz (GHz) or faster with two or more cores
* At least 16 GB of RAM

Install Procedure
~~~~~~~~~~~~~~~~~

Before you install Dissect it is recommended to create a `virtual environment <https://docs.python.org/3/tutorial/venv.html>`_.

.. code-block:: console

    $ pip install dissect

> That's all, you're now ready to use Dissect. To get you started, grab a test forensic image `here <https://cfreds.nist.gov/>`_.

Reviewer: ```suggestion
That's all, you're now ready to use Dissect. If you want to try it out, grab a test forensic image `here <https://cfreds.nist.gov/>`_.
```
